packages/components/pie-input/src/index.ts

```diff
@@ -171,6 +183,7 @@ export class PieInput extends FormControlMixin(RtlMixin(LitElement)) implements
                     @change=${this.handleChange}
                     data-test-id="pie-input">
                 <slot name="trailing"></slot>
+                ${assistiveText ? html`<pie-assistive-text variant=${status || nothing} data-test-id="pie-input-assistive-text">${assistiveText}</pie-assistive-text>` : nothing}
```

Comment:
I think it's more appropriate to use `ifDefined` for prop values, something like this:
(I haven't tested this, just did it free-hand)
```suggestion
                ${assistiveText ? html`<pie-assistive-text variant=${ifDefined(status)} data-test-id="pie-input-assistive-text">${assistiveText}</pie-assistive-text>` : nothing}
```